app/assets/javascripts/campaign.js

```diff
@@ -34,4 +34,46 @@ $(function() {
 			alert(response.error_description);
 		}
 	});
-});
\ No newline at end of file
+});
+
+$(function()
+{
+    $("#cc_payment").click(function(e)
+    {
+        $("#CCNUMBER").slideDown();
+        $("#EXPDATE").slideDown();
+        $("#donation-button").prop("disabled", false);
+        $("#bank_popup").show();
+        $("#cc_payment").hide();
+
+    });
+});
+
+$(function() {
+    WePay.set_endpoint("stage");
```

Comment:
We're calling this twice now so just call it once inside the $(function block at the top.